city_scrapers/spiders/newnj_nmc.py

```diff
@@ -0,0 +1,38 @@
+from city_scrapers_core.constants import CITY_COUNCIL
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import LegistarSpider
+
+
+class NewnjSpider(LegistarSpider):
+    name = "newnj_nmc"
+    agency = "Newark Municipal Council"
+    timezone = "America/Chicago"
+    start_urls = ["https://newark.legistar.com/Calendar.aspx"]
+    location = {
+        "address": "City Hall, 920 Broad Street, Newark, NJ 07102",
+        "name": "Municipal Council Chamber",
+    }
+
+    def parse_legistar(self, events):
+        """
+        `parse_legistar` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for event in events:
+            meeting = Meeting(
+                title=event["Name"]["label"],
+                description="",
+                classification=CITY_COUNCIL,
+                start=self.legistar_start(event),
+                end=None,
+                all_day=False,
+                time_notes="",
+                location=self.location,
```

Comment:
When I looked at the meeting list from [this page](https://newark.legistar.com/Calendar.aspx), I noticed that there are a few old meetings (from 2022) not having location of the Council Chamber. For example:
- Council Conference Room Third Floor: https://newark.legistar.com/MeetingDetail.aspx?ID=1000983&GUID=CF4888A4-D29E-415D-9B5C-F22C0AC931DC&Options=info|&Search=
- New Jersey Performance Art Center: https://newark.legistar.com/MeetingDetail.aspx?ID=1000978&GUID=C0690C91-F184-44D5-A38D-DFC12D21BD9F&Options=info|&Search=

I'm curious if there will be future meetings with different locations and if it's worth handling those cases. Thus, I've devised another approach for this field:
- If `item["Meeting Location"]` is "Council Chamber", we can use your detailed address.
- Otherwise, we should retain the original value of `item["Meeting Location"]`.

What do you think about these cases?

Reply:
I think this is an excellent suggestion and I appreciate your thoroughness, @lamle-ea. Thanks for looking so closely at the agency's site. I've now implemented this 👍